Jint/Native/Array/ArrayPrototype.cs

```diff
@@ -1290,5 +1234,67 @@ public JsValue Pop(JsValue thisObject, JsValue[] arguments)
             o.SetLength(len);
             return element;
         }
+
+        private sealed class ArrayComparer : IComparer<JsValue>
```

Comment:
I'd argue it's more common to have `sort` called without callback, so might be beneficial to have separate version of that kind of compare which has no state and could use as `ArrayComparer.Instance`

Reply:
I agree